openid-4-verifiable-presentations-1_0.md

```diff
@@ -1704,6 +1704,17 @@ Note: If the Verifier's Response URI did not return a `redirect_uri` in step (6)
 
 (10) The Verifier checks whether the `nonce` received in the Credential(s) in the VP Token in step (9) corresponds to the `nonce` value from the session. The Verifier then consumes the VP Token and invalidates the `transaction-id`, `request-id` and `nonce` in the session.
 
+## Pre-Final Specifications
+
+Implementers should be aware that this specification uses several specifications that are not yet final specifications. Those specifications are:
+
+- OpenID Federation 1.0 draft -42 [@!OpenID.Federation]
+- SIOPv2 draft -13 [@!SIOPv2]
+- SD-JWT-based Verifiable Credentials (SD-JWT VC) draft -08 [I-D.ietf-oauth-sd-jwt-vc]
+- W3C Digital Credentials API - [@!W3C.Digital_Credentials_API]
```

Comment:
This is a slightly odd one - I'm not sure the "only use the specifically referenced draft versions" really applies here as really your only option is to use whatever the browsers/OS actually implement?

Reply:
@leecam do you have an opinion here?

Reply:
Yeah DC API felt really weird to me as well and there isn't really a specific draft to reference?

Reply:
WG discussion remove this from here

Reply:
```suggestion
```